packages/web-player-component/src/index.js

```diff
@@ -15,22 +18,50 @@ export default class PlayerComponent extends HTMLElement {
     //Create wrapper and attach to DOM
     const wrapper = document.createElement('div');
     this.appendChild(wrapper);
-    //Init player with wrapper
-    this.player = webplayer(wrapper);
-  }
-  attributeChangedCallback() {
-    let attributeValue = this.getAttribute('source');
-    this.player.load(attributeValue).then(() => {
-      this.player.play()
+    //Create video element and attach to DOM
+    this.video = document.createElement('video');
+    wrapper.appendChild(this.video);
+    //Init player and skin
+    this.player = new WebPlayer({ video: this.video });
+    renderEyevinnSkin({
+      root: wrapper,
+      player: this.player,
+      castAppId: {}
     });
   }
+  
+  //Handles changed attributes of player-component. Changes video attributes directly (not through player functions).
+  attributeChangedCallback(name) {
+    console.log(name);
+    if (name === 'source') {
+      this.player.load(this.getAttribute('source'));
+    }
+    if (name === 'muted') {
+      if (this.getAttribute('muted') === 'true') {
+        this.video.muted = true;
+      }
+      else if (this.getAttribute('muted') === 'false') {
+        this.video.muted = false;
+      }
+    }
+    if (name === 'autoplay') {
+      if (this.getAttribute('autoplay') === 'true') {
+        this.video.setAttributeNode(document.createAttribute('autoplay'));
```

Comment:
isn't it enough to just use `setAttribute`?

Reply:
This works if we set 'autoplay'='autoplay' (can't be true/false)